easypy/concurrency.py

```diff
@@ -601,7 +601,7 @@ def __init__(self, items=None, log_ctx=None, workers=None, initial_log_interval=
 
     @property
     def L(self):
-        return list(self)
+        return self._items
```

Comment:
I'm actually concerned about this - one would expect `.L` to return a **L**ist. How about keeping `.L` and adding `.T` for retrieving the tuple?

Reply:
I agree, though we'd want to search&replace L->T because normally we don't need it as a mutable list.